modules/apps/subscription/subscription-service/src/main/java/com/liferay/subscription/service/impl/SubscriptionLocalServiceImpl.java

```diff
@@ -156,6 +161,24 @@ public Subscription addSubscription(
 			}
 		}
 
+		if (className.equals(AssetTag.class.getName())) {
```

Comment:
@velosofelipe64 this same code (green) between 164 - 181 is applied from 228 - 245, it could be a function and be reused?

I don't know if it's the best way, but should help avoiding duplicating code

What you think, @joseabelenda , @nNilton ??